tests/integrations/github/cicd/test_integration.py

```diff
@@ -1581,14 +1560,7 @@ def test_overlapping_changes_models(
  LEFT JOIN current_marketing AS m
    ON o.customer_id = m.customer_id
 ```
-
-```
-
-Directly Modified: sushi.customers (Non-breaking)
```

Comment:
Why did the output change so drastically?

Reply:
I tried to cover this with the PR description. I used to use the output that we had for categorization in order to get this nice direct change -> indirect relationship but switching to this other console method loses that. The problem was that the other method was only designed to help with categorization so it was missing things like removed models.